### DISCUSSION TITLE
Keep Empty Value Display Column

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#10344] [Help] [answered]
Hello, 
I try to set format date value, but when there is no data, column displays the format date value. How to keep the column displays empty when there is no data.

![Screenshot_20230419_030257](https://user-images.githubusercontent.com/5665115/232892657-86a834d0-9cff-4ff0-bb04-ff7b0132090c.png)

**Comment by andz-bb** [comment, marked as answer]
hey @muksidin 

You can wrap the Handlebars expression in an if statement to only format the column value if it exists. I used this and it works well on my setup:

```hbs
{{#if (isTruthy Value)}}
  {{date Value "DD-MM-YYYY"}}
{{/if}}
```

When attempting more complex logic in bindings I often find it more intuitive to switch over to the JS functions instead. You can make use of the same helpers from Handlebars in the function as well which can save a lot of time writing custom JS. Here's an equivalent function I had initially written for this:

```js
return !!$("Value") ? helpers.date($("Value"), "DD-MM-YYYY") : ""
```

Hope you find these useful!

**Reply by muksidin** [reply]
Thanks alot @andz-bb for your quick help :)
